mpxe/integration_tests/test_can_dump.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import time
+
+from mpxe.integration_tests import int_test
+
+class TestCanDump(int_test.IntTest):
+    def setUp(self):
+        super(TestCanDump, self).setUp()
+        self.can_dump = self.manager.start('can_dump')
+
+    def test_can_dump(self):
+        time.sleep(1)
```

Comment:
This test confuses me, was this just to test that project/pm works? Should we keep this? It is just adding a second to testing time for no reason, perhaps shorten the delay significantly?